### DISCUSSION TITLE
Multiple lavalink

### DISCUSSION THREAD

**Discussion by rekayno** [SudhanPlayz/Discord-MusicBot#474] [Ideas]
I would like the bot to use additional servers with a heavy load on the main lavalink server.

**Comment by testbot-github** [comment]
Yea

**Comment by testbot-github** [comment]
https://github.com/SudhanPlayz/Discord-MusicBot/projects/3#card-70945730

> Done/Already works with new version

**Reply by testbot-github** [reply]
@rekayno

**Comment by DarrenOfficial** [comment]
Coming in v5.
